### PR TITLE
Add message activities

### PR DESCRIPTION
**Pull request by benlovell**
Gives the booking manager the ability to leave messages against a
booking request and / or an appointment.

There is no validation and the booking manager can leave a blank
message if they like.

### CODE REVIEW

File: app/assets/javascripts/poller.js [hunk 1]
```diff
@@ -36,7 +37,11 @@
     },
 
     insertActivity: function(text) {
-      $(text).hide().prependTo(this.$poller).fadeIn();
+      var element = $(text);
+
+      if (this.$poller.find('#' + element.attr('id')).length == 0) {
```
Comment (benlovell) [inline app/assets/javascripts/poller.js]: Happy to take a steer on something more concise here.

**Comment by benlovell** [comment]
This is good for review now. @benbarnett @Guntrisoft

File: app/assets/javascripts/poller.js (hunk 1 above)
Comment (benbarnett) [inline app/assets/javascripts/poller.js]: As id's are unique to the whole page you could probably just use `getElementById` and then boolean-ise it. Does this seem readable?

```js
if (!!!document.getElementById(element.attr('id')) {
```

Comment (benlovell) [inline app/assets/javascripts/poller.js]: Nicely.

**Comment by Guntrisoft** [comment]
Only observation is you can submit a blank message but as discussed we'll see if someone does do that - otherwise looking good to me 👍 ✅

**Pull request merged**